server/auth/oidc.go

```diff
@@ -61,7 +61,11 @@ func ExchangeCode(ctx context.Context, r *http.Request, config *oauth2.Config, p
 
 	oauth2Token, err := config.Exchange(ctx, r.URL.Query().Get("code"))
 	if err != nil {
-		return nil, echo.NewHTTPError(http.StatusInternalServerError, "Unable to exchange token: "+err.Error())
+		// Return error description from Auth0 query param or static message?
+		return nil, echo.NewHTTPError(http.StatusInternalServerError, r.URL.Query().Get("error_description"))
```

Comment:
imo we should keep the original generic message + err.Error(), then append `error_description` as this is Auth0 specific and not all auth providers will have it in an URL query